docs/quick_start.rst

```diff
@@ -84,135 +113,248 @@ on multiple inputs.
+    the functions ``collect_successful_results()`` and ``open_output_file()``)
+    to gather all results.
+
     .. code-block:: bash
 
-        # Here, we use the popular pandas package to aggregate the results.
-        # Make sure that pandas is installed on your machine (pip3 install pandas).
-        $ ./eval.py
+        $ python3 eval.py
 
-    ::
+    Output:
 
-        experiment      comparisons          swaps      time
-        bubble-sort        499500.0  253437.333333  0.091776
-        insertion-sort     241891.0  257609.000000  0.039501
+    ::
 
-    ``eval.py`` is a simple 25 line script that uses the simexpal Python interface
-    (i.e., the functions ``collect_successful_results()`` and ``open_output_file()``)
-    to gather all results. It uses ``pandas`` to aggregate statistics over all experiments.
+        comparisons          swaps      time
+        experiment                                          
+        bubble-sort        499500.0  253437.333333  0.053750
+        insertion-sort     241891.0  257609.000000  0.027219
 
 .. tip::
     Simexpal supports autocomplete via `argcomplete <https://pypi.org/project/argcomplete/>`_.
     To enable autocomplete, install ``argcomplete`` and enable global completion:
 
     .. code-block:: bash
-
+      
       $ pip3 install argcomplete
       $ activate-global-python-argcomplete
 
 Running Experiments
 -------------------
 
-As a simple example, we compare Insertion Sort and Bubble Sort on a set of instances
-(such an example is available in the  within `examples
-<https://github.com/hu-macsy/simexpal/tree/master/examples>`_).
-To this purpose, we created a new `sorting
-<https://github.com/hu-macsy/simexpal/tree/master/examples/sorting>`_
-directory, and wrote a short Python script
-`sort.py
-<https://github.com/hu-macsy/simexpal/blob/master/examples/sorting/sort.py>`_
-where we implemented the two algorithms.
-``sort.py`` accepts two arguments: the algorithm name (i.e. insertion-sort or bubble-sort)
-and the path to the instance.
-Then, you generated a bunch of instances and placed them in a directory called "instances"
-within "project" (in this example we just deal with a single instance
-`random_500.list
-<https://github.com/hu-macsy/simexpal/blob/master/examples/sorting/instances/random_500.list>`_).
-Now we can run a sorting algorithm on a specific instance with:
+Let us now take a closer look at running experiments. As an example, we compare
+the sorting algorithms *insertion sort* and *bubble sort* using a set of
+instances. Please find the resources in our `example folder
+<https://github.com/hu-macsy/simexpal/tree/master/examples>`_. 
+
+If you look at the `sorting directory
+<https://github.com/hu-macsy/simexpal/tree/master/examples/sorting>`_ in our
+examples, you will find `my_sort.py
+<https://github.com/hu-macsy/simexpal/blob/master/examples/sorting/my_sort.py>`_
+including an implementation of the two algorithms *insertion sort* and *bubble
```

Comment:
`my_sorting_algorithms.py` implements merge-sort, as well.

Reply:
I would leave this still as is, because we do not want to introduce the reader to the *merge sort* algorithm yet. Therefore, I would not overload the reader with more information than needed, even though the file contains one more algorithm.